survey_legal/views/survey_user_input_views.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="survey_user_input_view_form" model="ir.ui.view">
+        <field name="model">survey.user_input</field>
+        <field name="inherit_id" ref="survey.survey_user_input_view_form" />
+        <field name="arch" type="xml">
+            <field name="user_input_line_ids" position="after">
+                <field name="survey_legal_terms" invisible="1" />
+                <group
+                    name="metadata"
+                    attrs="{'invisible': ['|', ('survey_legal_terms', '!=', True), ('state', '=', 'new')]}"
```

Comment:
Isn't better to simply show this if filled?

Reply:
Yes, it would TBH :sweat_smile: